Frameworks/include/CGContextCairo.h

```diff
@@ -138,7 +138,7 @@ class CGContextCairo : public CGContextImpl {
     virtual bool CGContextIsPointInPath(bool eoFill, float x, float y);
     virtual CGPathRef CGContextCopyPath(void);
 
-    virtual void CGContextDrawGlyphRun(const DWRITE_GLYPH_RUN* glyphRun);
+    virtual void CGContextDrawGlyphRun(const DWRITE_GLYPH_RUN* glyphRun, float lineHeight);
```

Comment:
Possibly name them lineAscent and lineDescent so it's clearer that they're metrics of the surrounding line.

---
In reply to: [84549518](https://github.com/Microsoft/WinObjC/pull/1200#discussion_r84549518) [](ancestors = 84549518)